.env.production

```diff
@@ -0,0 +1,8 @@
+DATABASE_NAME=answerking_app
+DATABASE_HOST=host.docker.internal
+DATABASE_PORT=3306
+DATABASE_USER=root
+DATABASE_PASS=Admin123
+SECRET_KEY="django-insecure-x977=v5a2q-e%_5$b9ge@jzk%s_nhk2l8_use&h@*m$w33dopr"
+DJANGO_SETTINGS_MODULE=answerking.settings.base
+DATABASE_ENGINE = "django.db.backends.mysql"
```

Comment:
why? use github secrets as you'd normally do, you might have to create a overriding it in the workflow using another variable `DJANGO_SETTINGS_MODULE = {{GITHUB.DJANGO_SETTINGS_MODULE_DEPLOY}}`

Reply:
this si when you are running it locally so it wouldn't have access to github secrets